src/index.ts

```diff
@@ -218,7 +219,8 @@ export const withSpid = ({
         metadataTamperer,
         getPreValidateResponse(
           serviceProviderConfig.strictResponseValidation,
-          appConfig.eventTraker
+          appConfig.eventTraker,
```

Comment:
small typo, this should be `tracker`

Reply:
should i do this fix in this PR or is it out of scope? @michaeldisaro